webapp/crashstats/supersearch/form_fields.py

```diff
@@ -72,8 +72,8 @@ def to_python(self, value):
 
         return super().to_python(value)
 
-    def clean(self, *args, **kwargs):
-        cleaned_value = super().clean(*args, **kwargs)
+    def clean(self, value):
```

Comment:
I fixed `clean` things to conform to the Django Field API. This reduces likelihood it breaks in a future Django upgrade.